### ISSUE TITLE
Reference to nonexistent source map in compiled output

### ISSUE THREAD

**Issue by nertzy** [pusher/pusher-js#535]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
I am using source-map-loader in my webpack configuration. When I load pusher via webpack, I get a warning that a source map failed to be parsed:

```
WARNING in ./node_modules/pusher-js/dist/web/pusher-with-encryption.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/dev/projects/syrup/node_modules/pusher-js/dist/web/utf8.js.map' file: Error: ENOENT: no such file or directory, open '/Users/dev/projects/syrup/node_modules/pusher-js/dist/web/utf8.js.map'
Error: Failed to parse source map from '/Users/dev/projects/syrup/node_modules/pusher-js/dist/web/utf8.js.map' file: Error: ENOENT: no such file or directory, open '/Users/dev/projects/syrup/node_modules/pusher-js/dist/web/utf8.js.
map'
    at fetchFromFilesystem (/Users/dev/projects/syrup/node_modules/source-map-loader/dist/utils.js:131:11)
    at async fetchFromURL (/Users/dev/projects/syrup/node_modules/source-map-loader/dist/utils.js:232:21)
    at async Object.loader (/Users/dev/projects/syrup/node_modules/source-map-loader/dist/index.js:69:9)
ℹ ｢wdm｣: Compiled with warnings.
```

This is because there is a source map comment left over in `dist/web/pusher.js` that appears to have been copied over from a third-party library that points to a file that is not present in `dist/`.

https://github.com/pusher/pusher-js/blob/8e57f4ec606901a17f98941f3a2273b80eff7bea/dist/web/pusher.js#L546

**What is the expected behavior?**
Source map comments should be stripped out when possible if they are no longer accurate.

**Which versions of Pusher, and which browsers / OS are affected by this issue?
Did this work in previous versions of Pusher? If so, which?**
- pusher-js 7.0.3
- webpack 4.46.0
- source-map-loader 1.1.3

**Comment by leesio** [comment]
Hey, thanks for reporting this. It looks like these are references from some dependencies we're bundling. I'll try and find a way to strip them when we bundle

**Comment by ebeigarts** [comment]
@leesio I'm still having this issue, can you reopen this ticket?

**Comment by leesio** [comment]
Hi there,
I'm afraid I no longer work for Pusher, so I won't be able to reopen the issue for you. You might get more luck contacting [pusher support](support@pusher.com).

**Comment by benw-pusher** [comment]
Thanks for responding, James! 
I've re opened this issue and will raise it internally for attention.

**Comment by simonbowen** [comment]
We're getting the same problem.

**Comment by cookie-for-pres** [comment]
the error is yet to be fixed and why is that?

**Comment by cookie-for-pres** [comment]
yeah i just stopped using pusher all together, just use socket.io it works the same and your not limited to your request.

**Comment by benw-pusher** [comment]
I have escalated this issue internally and we will update here when a resolution is available.

**Comment by c0derdx** [comment]
Not fixed i faced this error today

**Comment by nvidda-hub** [comment]
I also faced this issue by today itself.

**Comment by marcelcorso** [comment]
Sorry for the really slow fix, everybody! 

After some discussion (thanks @CoolGoose and @ d7my11) 
 @aodinok tried to 

* enable source map generation in pusher-js with devtool: true
* and use source-map-loader to consume existing sourcemaps from 3rd party libraries and put them to generated source map

on https://github.com/pusher/pusher-js/pull/570

@nertzy  @nvidda-hub  @c0derdx do you think this will work for you?

**Comment by benw-pusher** [comment]
Fixed by release 7.0.4

**Comment by nvidda-hub** [comment]
Thanks man
